Engine_Revit_UI/Query/PolyCurve.cs

```diff
@@ -237,6 +237,7 @@ internal static PolyCurve PolyCurve(this FamilyInstance familyInstance, HostObje
             return PolyCurve(familyInstance, aSolidList, aPlaneList, pullSettings);
         }
 
+
```

Comment:
Was this blank line needed?

Reply:
Same as above: I usually leave 2 blanks before the header.